lib/src/react16_suggestors/react16_utilities.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:analyzer/analyzer.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:source_span/source_span.dart';
+
+import 'constants.dart';
+
+/// Returns whether or not the source file contains the React 16 validation
+/// required comment.
+///
+/// Can be used to determine whether or not the file has been modified
+/// already. This is useful (in combination with [allComments] because the
+/// visitor used for visiting comments does not work as expected, making it
+/// difficult to iterate over comments.
+bool hasValidationComment(AstNode node, SourceFile sourceFile) {
```

Comment:
These utilities were moved in the codemod branch (#36), and test coverage for both functions was added as well.

Reply:
It shouldn't cause conflicts because they'll just exist in two places. They didn't exist in #36 at all, so as long as the file doesn't import both the global utilities and react16_utilities, there shouldn't be any issue except the redundancy.

I was going through it with Keal yesterday and it seemed easiest to just wait for this to merge, pull it into the component2 branch, and then address it. Would you rather we just do that now though, @aaronlademann-wf?